docs/UserGuide.md

```diff
@@ -299,7 +324,24 @@ Examples:
 * `c-find John` returns `john` and `John Doe`.
 * `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
 
-#### 4.5 Deleting a person : `c-delete`
+#### 4.7 Locating persons based on matching tags: `c-tag-find`
+
+Finds all contacts that contain the KEYWORD(s) in their tags.
+
+Format: `c-tag-find KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `Friday` will match `friday`.
+* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
+* Only the tag names are searched.
+* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
+* Employees whoes tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
+ (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+
+Examples:
+* `c-tag-find friday` returns all employees whose tags contain `friday`.
+* `c-tag-find friday parttime` returns all employees whose tag contain `friday`, `parttime`.<br>
```

Comment:
Maybe tag could be be "tags" here?

Reply:
Thank you! I've changed it in the latest commit.